org.jacoco.build/pom.xml

```diff
@@ -776,12 +809,14 @@
       </build>
     </profile>
 
+    <!-- -->
```

Comment:
Empty comment?

Reply:
Yep - for visual separation of profiles that follow this comment from ones that are above.